src/config/CemuConfig.cpp

```diff
@@ -1,23 +1,20 @@
 #include "config/CemuConfig.h"
+#include "WindowSystem.h"
 
 #include "util/helpers/helpers.h"
 #include "config/ActiveSettings.h"
 
-#include <wx/language.h>
-
 #include "ActiveSettings.h"
 
-XMLCemuConfig_t g_config(L"settings.xml");
-
 void CemuConfig::SetMLCPath(fs::path path, bool save)
 {
 	mlc_path.SetValue(_pathToUtf8(path));
 	if(save)
-		g_config.Save();
+		GetConfigHandle().Save();
 	Account::RefreshAccounts();
 }
 
-void CemuConfig::Load(XMLConfigParser& parser)
+XMLConfigParser CemuConfig::Load(XMLConfigParser& parser)
```

Comment:
I couldn't really make sense of this change. Why are `CemuConfig::Load` and `CemuConfig::Save` returning a copy of the `XMLConfigParser` thats passed by reference?

Reply:
The intent was to return a parser for a subtree XML element where child settings would read/store their settings.
And `CemuConfig` returns a parser for the `content` XML element instead of the original parser.
```
XMLConfigParser CemuConfig::Load(XMLConfigParser& parser)
{
	auto new_parser = parser.get("content");
	if (new_parser.valid())
		parser = new_parser;

	// ...

	return parser;
}

XMLConfigParser CemuConfig::Save(XMLConfigParser& parser)
{
	auto config = parser.set("content");

	// ...

	return config;
}
```

Reply:
Ah I didn't see the reassignment. Thanks